src/meter/Meter.d.ts

```diff
@@ -0,0 +1,32 @@
+import { CSSProperties, PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export namespace MeterProps {
+  type Range = {
+    startValue: number,
+    endValue: number,
+    fillingColor?: string,
+    labelColor?: string,
+    backgroundColor?: string
+  }
```

Comment:
I'ld add a TODO because of it was a refinement with `r => r.startValue < r.endValue`